catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -165,14 +172,38 @@ public void setResultShow(List<String> resultShow) {
 
     public void setAttributeAliases(List<String> attributeAliases) {
         this.attributeAliases = attributeAliases.stream()
-                .map(str -> str.split("="))
+                .map(str -> str.split("=", 2))
+                .filter((list) -> {
+                    if (list.length <= 1) {
+                        if (LOGGER.isDebugEnabled()) {
```

Comment:
I added the check on @rzwiefel's advice - " probably a good idea to wrap it in a LOGGER.isDebugEnabled() since this is inside something being streamed"

Reply:
Oh, I see that comment. @rzwiefel, can you explain your advice?

Reply:
Since this is something that is being looped over (it is in a stream) it could (theoretically) have a large number of items in it, and in that case we would be logging to debug **N** number of times ... And relatively an if statement is much cheaper than the log statement. So we only actually pay the cost of logging it if the logs are set at that level.

Reply:
Note: as per online discussion, this `LOGGER.isDebugEnabled()` check can be removed.